widgetastic_manageiq/__init__.py

```diff
@@ -2713,65 +2714,110 @@ def ProviderEntity():  # noqa
 
 
 class DashboardWidgetsPicker(View):
-    """ Represents widgets picker in Dashboard editing screen (Cloud Intel/Reports/Dashobards).
+    """Represents widgets picker in Dashboard editing screen (Cloud Intel/Reports/Dashobards).
```

Comment:
Can you fix the `Dashobards` typo? Unless there is a bard singing a song for Dasha in real :)